benches/end2end.rs

```diff
@@ -64,7 +64,7 @@ fn end2end_benchmark(c: &mut Criterion) {
 
     group.bench_with_input(benchmark_id, &size, |b, &s| {
         b.iter(|| {
-            let ptr = go_base::<pasta_curves::pallas::Scalar>(&mut store, s.0, s.1);
```

Comment:
You can chuck the following in a `clippy.toml` at the root of the repo to have the yelling done by tooling instead of you:
```
disallowed-methods = [
    # we use strict naming conventions for fields of our workhorse curve
    { path = "pasta_curves::Fp", reason = "use pasta_curves::pallas::Scalar instead" },
    ...
]
```

Reply:
Thanks. However, please note that the code you provided here *reproduces the error I sought to correct here*. In fact, `pasta_curves::Fq` is the one that corresponds to `pallas::Scalar` — as noted in the added documentation.

I realize this isn't what you were intending to communicate, and I mention it only because:
- It's still wrong…
- It's a *perfect* and timely example of how allowing these wrong expressions into our source of truth creates ongoing confusion.

As always, I recognize that I did approve the PR(s) introducing these errors.